dp/jobqueueing/alembic/README

```diff
@@ -0,0 +1 @@
+Generic single-database configuration.
```

Comment:
Probably should move some or all of the explanation of multi-database configuration out of env.py into here.